ui/apps/platform/src/Containers/Integrations/Banners/OcmDeprecatedToken.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { ReactElement } from 'react';
+import { Alert, PageSection } from '@patternfly/react-core';
+import { useSelector } from 'react-redux';
+
+import { selectors } from 'reducers';
+
+function ocmDeprecatedCounter(integrations: { type: string; credentials: { secret: string } }[]) {
```

Comment:
Is it possible to use the integration type here instead of inline type?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/services/CloudSourceService.ts#L17

Although src/reducers/cloudSources.js is not typed, I do not see any ui compile or lint among failures.